### DISCUSSION TITLE
Details on Tampermonkey Legacy support in Chrome after Manifest V3 transition

### DISCUSSION THREAD

**Discussion by sabelliadriano** [Tampermonkey/tampermonkey#2289] [General]
Hello Tampermonkey team,

Can I have more information about how the Manifest V3 transition will affect Tampermonkey Legacy specifically:

 What is the exact date when Tampermonkey Legacy will cease to function in Chrome due to the Manifest V2 deprecation?

 Will Tampermonkey Legacy be automatically removed from Chrome, or will it simply stop working on that date?

How will the Chrome Web Store handle Tampermonkey Legacy as we approach the cut-off date? Will it be delisted or marked as incompatible at some point?

Is there a specific time on the final day when Tampermonkey Legacy will stop working, or will it be a gradual process?

Will users of Tampermonkey Legacy receive any in-browser notifications or warnings as the cut-off date approaches?

 Given these timelines, how urgently do you recommend we complete our migration from Legacy to version 5.3?

Thank you for your assistance and for your continued work on Tampermonkey.

Best regards,
Adriano

**Comment by derjanb** [comment]
Hi!

most of the requested information depends on how Google is going to handle it.

> What is the exact date when Tampermonkey Legacy will cease to function in Chrome due to the Manifest V2 deprecation?

Please see https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline

> Will Tampermonkey Legacy be automatically removed from Chrome, or will it simply stop working on that date?
> Is there a specific time on the final day when Tampermonkey Legacy will stop working, or will it be a gradual process?

Tampermonkey Legacy will continue to work in older versions of Chrome. However, Google might drop support for these versions at newer Chrome versions.

> How will the Chrome Web Store handle Tampermonkey Legacy as we approach the cut-off date? Will it be delisted or marked as incompatible at some point?

It's already marked as "This extension may soon no longer be supported because it doesn't follow best practices for Chrome extensions." on the Chrome Web Store. Delisting is probable at some point.

> Will users of Tampermonkey Legacy receive any in-browser notifications or warnings as the cut-off date approaches?

Users already receive warnings in the Web Store and at Chrome's extension page.

> Given these timelines, how urgently do you recommend we complete our migration from Legacy to version 5.3?

The earlier the better. The migration is quite easy and should not take more than a few minutes. If bugs are found, please report them so that they can be fixed before Tampermonkey Legacy stops working.

**Comment by sergeevabc** [comment]
Setup: Windows 7, Edge 109.
Which Tampermonkey version is for me then?
